### DISCUSSION TITLE
NEW model vicuna-13B-1.1-GPTQ-4bit-128g

### DISCUSSION THREAD

**Discussion by Felixqian4160** [oobabooga/text-generation-webui#1438] [General]
i ues this model but GIBBERISH OUTPUT？
how to update GPTQ-for-LLaMA.safetensors？
https://huggingface.co/TheBloke/vicuna-13B-1.1-GPTQ-4bit-128g

**Comment by Ph0rk0z** [comment]
download the one without act-order

**Comment by syddharth** [comment]
Need to update \text-generation-webui\repositories\GPTQ-for-LLaMa to the latest to be able to use 4bit quantized models.

**Reply by Felixqian4160** [reply]
I allready update these,but  GIBBERISH OUTPUT as well
safetensors==0.3.0 
datasets==2.11.0
sentencepiece==0.1.98
transformers==4.28.1
accelerate==0.18.0
texttable==1.6.7

**Comment by snicolast** [comment]
I use this version and works fine with the "old" GPTQ-for-LLaMa. No need to update it.

https://huggingface.co/tsumeone/vicuna-13B-1.1-GPTQ-4bit-128g-cuda

**Reply by Felixqian4160** [reply]
Thank you very much for your help. This is very helpful.

**Reply by mikolodz** [reply]
I'm using a safetensors version with triton and it works pretty well, although I can't make it work with LORA and monkey-patch.
Anyone trained 4b Vicuna with some results?
